src/ngraph_builder.cc

```diff
@@ -119,7 +119,71 @@ static tf::Status TranslateUnaryOp(tf::Node* op, Builder::OpMap& ng_op_map) {
   return TranslateUnaryOp(op, ng_op_map, [](std::shared_ptr<ng::Node> n) { return make_shared<T>(n); });
 }
 
-// Helper for Builder::TranslateGraph (elementwise binops)
+// Helper function to translate a binary op
```

Comment:
expand this sentence to something like: "that requires combination of multiple nGraph ops"

Reply:
This function can be used for any binary op. I made changes in the other TranslateBinaryOp() function, to remove the redundant code